src/chains.rs

```diff
@@ -202,7 +202,7 @@ impl ChainItemKind {
     fn is_tup_field_access(expr: &ast::Expr) -> bool {
         match expr.kind {
             ast::ExprKind::Field(_, ref field) => {
-                field.name.to_string().chars().all(|c| c.is_digit(10))
+                field.name.to_string().bytes().all(|c| c.is_ascii_digit())
```

Comment:
Why chars to bytes? Seems not equivalent :eyes:

Reply:
It's true that it is unrelated. But if all the chars are ASCII digits, then none of them are multi-byte, so it is equivalent